sparrowpy/classes/RadiosityFast.py

```diff
@@ -358,6 +358,80 @@ def from_polygon(
             patches_points, n_patches,
             patch_to_wall_ids)
 
+    @classmethod
+    def from_file(cls, filename: str, manual_patch_size=None,
+                       auto_walls=True):
+        """Create a Radiosity object from a blender or stl file.
+
+        Parameters
+        ----------
+        filename: string
```

Comment:
```suggestion
        filepath: string
```
technically its not the name but the path